plasmapy/formulary/shocks.py

```diff
@@ -0,0 +1,122 @@
+""" This module gathers functions relating to shocks and the properties thereof.
+"""
+__all__ = ["entropy_jump_polytropic"]
+
+
+from astropy import units as u
+from astropy.constants import c, a0, k_B
+from numpy import pi, exp, sqrt, log, nan, isclose
+from warnings import warn
+
+from plasmapy.utils.exceptions import PhysicsWarning
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    check_relativistic,
+    validate_quantities,
+)
+
+
+@validate_quantities(
+    rho_1={"can_be_negative": False, "pass_equivalent_units": True},
+    rho_2={"can_be_negative": False, "pass_equivalent_units": True},
+    p_1={"can_be_negative": True, "pass_equivalent_units": True},
+    p_2={"can_be_negative": True, "pass_equivalent_units": True},
+    c_v={"can_be_negative": False},
+)
+def entropy_jump_polytropic(
+    c_v: u.J / u.K,
```

Comment:
Personally I rarely know the specific heat of the plasmas I'm dealing with, so using this function would always require some extra leg work. I would definitely suggest substituting Eq. 4.26 in Drake (which is equivalent to Eq. 3.3) to eliminate c_v, then replacing this input with a `plasmapy.Particle`  instance that represents the ion species to get the atomic number and charge. You can then present Eq. 4.24 and Eq. 4.26 in the documentation to make clear what's happening.

Reply:
I would keep `c_v` as is because people could use different methods to calculate it.  Instead, we should add a separate function (in a separate PR) to calculate `c_v`.  In the 2006 version of drake `c_v` is given in eqn 3.5.

Reply:
Are there different methods that would be valid under this polytropic gas assumption? I haven't been able to find much information on this outside of Drake. Unless there are obvious, well-known models (like there are for resistivity, for example), I think that at the very least the documentation for this function should provide an appropriate equation or point to at least one function in PlasmaPy that will calculate it.